velox/dwio/dwrf/test/ReaderTest.cpp

```diff
@@ -96,9 +91,24 @@ class TestReaderP
   std::unique_ptr<ExecutorBarrier> barrier_;
 };
 
+class TestReader : public testing::Test, public VectorTestBase {
+ protected:
+  std::vector<VectorPtr> createBatches(
+      const std::vector<std::vector<int32_t>>& integerValues) {
```

Comment:
integerValues -> values; it is obvious these are integers because the type is int32_t